### PR TITLE
Subscription Site: Polishing stuff before the release

### LINKED ISSUE #36021: Subscription Site: Polishing Subscription block after the post placement
Related to peKye1-EA-p2

We would like to release the Subscription block after the post placement toggle. Here is a list of things that still need to be done or reconsidered:

- [x] prevent the Subscribe block from appearing when the Paywall block is visible https://github.com/Automattic/jetpack/pull/36045
- [x] render the Subscribe block before the Share / Like blocks https://github.com/Automattic/jetpack/pull/36240
- [x] add some nudge p1HpG7-rgv-p2#comment-70287 before the block (design needed, cc @crisbusquets) p1HpG7-rgv-p2#comment-70289 https://github.com/Automattic/jetpack/pull/36107
- [x] consider removing the nudge for classic themes since users won't be able to edit it (we decided to leave it there p1709283358476459-slack-C052XEUUBL4)
- [x] polish the Jetpack Newsletter Settings to look good with only one toggle (design needed, cc @crisbusquets) p1HpG7-rrt-p2#comment-70336 https://github.com/Automattic/jetpack/pull/36177
- [ ] Automattic/wp-calypso#88462
- [x] wrap static texts with `__` when we have a final copy https://github.com/Automattic/jetpack/pull/36107 https://github.com/Automattic/jetpack/pull/36177
- [x] add support for inherit anchor bock layout https://github.com/Automattic/jetpack/pull/36107
- [x] add a link to the single post template editor for block themes https://github.com/Automattic/jetpack/pull/36218
- [x] remove custom styles from the `<h2>` element (and make it `<h3>` instead?) https://github.com/Automattic/jetpack/pull/36107#issuecomment-1981464914 https://github.com/Automattic/jetpack/pull/36240

### LINKED ISSUE #36342: Subscription Site: Improve toggle copy
### Impacted plugin

Jetpack

### What

Improve copy so they're more concise and easy to skim.

### Current implementation
https://github.com/Automattic/jetpack/pull/36218

image

### Proposal

- Allow visitors to subscribe to your site
- Add the Subscribe Block at the of each post. Preview and edit
- Subscription pop-up when scrolling a post. Preview and edit
- Enable the “subscribe to site” option on your comment form
- Enable the “subscribe to comments” option on your comment form

Screenshot 2024-03-12 at 12 39 08

### PR DESCRIPTION
**Pull request by pkuliga**
Fixes https://github.com/Automattic/jetpack/issues/36021
Fixes https://github.com/Automattic/jetpack/issues/36342

## Proposed changes:

It:
- improves the Newsletter settings toggle labels
- inherits font style, font size, and font weight from the theme for the Subscribe block header instead of using custom styles
- uses `<h3>` instead of `<h2>` for the Subscribe block header
- renders the Subscribe block after the sharing blocks for classic themes to be consistent with block ones

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

Make sure the mentioned changes are sane.

### CODE REVIEW

**Review by simison — APPROVED** [review]

File: projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx [hunk 1]
```diff
@@ -158,10 +160,7 @@ function SubscriptionsSettings( props ) {
 									onChange={ handleSubscribePostEndToggleChange }
 									label={
 										<>
-											{ __(
-												'Enable automatic insertion of the Subscribe block into the theme at the end of each post',
-												'jetpack'
-											) }
+											{ __( 'Add the Subscribe Block at the of each post', 'jetpack' ) }
```
Comment (jeherve) [inline projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx:163]: Popping in here, I noticed we're missing a word in that sentence!

Comment (pkuliga) [reply projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx:163]: @jeherve can you be more specific? Which word is missing and where? 😄 

When I'm looking at it now I have also doubts about "Subscribe block" vs "Subscribe Block".

Comment (jeherve) [reply projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx:163]: "at the **end** of each post"

Comment (pkuliga) [reply projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx:163]: Ah, I'm blind apparently, thanks!

Comment (pkuliga) [reply projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx:163]: PR with the fix: https://github.com/Automattic/jetpack/pull/36384

**Pull request merged**